core/src/test/scala/integration/kafka/server/KRaftClusterTest.scala

```diff
@@ -1579,6 +1579,36 @@ class KRaftClusterTest {
       cluster.close()
     }
   }
+
+  @Test
+  def testReduceNumNetworkThreads(): Unit = {
```

Comment:
Could you please rename it to `testIncreaseNumNetworkThreads` by virtue of recent change?